### PR TITLE
Fix for the #78

### PR DESCRIPTION
**Pull request by wakr**
This pull request will fix the .jsons and tests which were pointing to production URLs. All .jsons should now follow the logic `example.com/{resource}/` and the `example.com` acts as a placeholder for the test address. This "test address" is usually localhost:{freeport}.

### CODE REVIEW

File: src/main/java/fi/helsinki/cs/tmc/core/domain/Exercise.java [hunk 1]
```diff
@@ -28,10 +28,10 @@
     private String checksum; //: "406f2f0690550c6dea94f319b2b1580c",
 
     @SerializedName("return_url")
-    private String returnUrl; //: "https://tmc.mooc.fi/staging/exercises/284/submissions.json",
+    private String returnUrl; //: "https://example.com/staging/exercises/284/submissions.json",
 
     @SerializedName("zip_url")
-    private String zipUrl; //": "https://tmc.mooc.fi/staging/exercises/284.zip",
+    private String zipUrl; //": "https://example.com/staging/exercises/284.zip",
```
Comment (jamo) [inline src/main/java/fi/helsinki/cs/tmc/core/domain/Exercise.java]: I think we could just drop these sample strings here... (no need to do it in this PR but you can if you wish so)

**Comment by jamo** [comment]
LGTM, nice work.

Please rebase this on master and let us know, so that we can merge it.

@ljleppan request for LGTM or comments

**Comment by ljleppan** [comment]
LGTM

**Comment by wakr** [comment]
Rebased :+1:

**Pull request merged**